### PR TITLE
Add documentation to wasm wrappers

### PR DESCRIPTION
**Pull request by TheQuantumPhysicist**

### CODE REVIEW

File: wasm-wrappers/src/lib.rs [hunk 1]
```diff
@@ -63,6 +63,7 @@ impl From<Network> for ChainType {
     }
 }
 
+/// Indicates whether a token an be frozen
```
Comment (OBorce) [inline wasm-wrappers/src/lib.rs:66]: token can

File: wasm-wrappers/src/lib.rs [hunk 2]
```diff
@@ -288,30 +328,46 @@ pub fn staking_pool_spend_maturity_block_count(current_block_height: u64, networ
...
+/// since that output's transaction is included the blockchain
 #[wasm_bindgen]
-pub fn encode_lock_for_block_count(num: u64) -> Vec<u8> {
-    let output = OutputTimeLock::ForBlockCount(num);
+pub fn encode_lock_for_block_count(block_count: u64) -> Vec<u8> {
+    let output = OutputTimeLock::ForBlockCount(block_count);
     output.encode()
 }
 
+/// Given a number of clock seconds, this function returns the the output timelock
```
Comment (OBorce) [inline wasm-wrappers/src/lib.rs:340]: the the

File: wasm-wrappers/src/lib.rs [hunk 3]
```diff
@@ -288,30 +328,46 @@ pub fn staking_pool_spend_maturity_block_count(current_block_height: u64, networ
...
+/// which is used in locked outputs to lock an output for a given number of seconds
+/// since that output's transaction is included the blockchain
```
Comment (OBorce) [inline wasm-wrappers/src/lib.rs:342]: included "in" the blockchain

File: wasm-wrappers/src/lib.rs [hunk 4]
```diff
@@ -288,30 +328,46 @@ pub fn staking_pool_spend_maturity_block_count(current_block_height: u64, networ
...
 #[wasm_bindgen]
-pub fn encode_lock_for_seconds(num: u64) -> Vec<u8> {
-    let output = OutputTimeLock::ForSeconds(num);
+pub fn encode_lock_for_seconds(total_seconds: u64) -> Vec<u8> {
+    let output = OutputTimeLock::ForSeconds(total_seconds);
     output.encode()
 }
 
+/// Given a timestamp represented by as unix timestamp, i.e., number of seconds since unix epoch,
+/// this function returns the the output timelock which is used in locked outputs to lock an output
```
Comment (OBorce) [inline wasm-wrappers/src/lib.rs:350]: the the

File: wasm-wrappers/src/lib.rs [hunk 5]
```diff
@@ -288,30 +328,46 @@ pub fn staking_pool_spend_maturity_block_count(current_block_height: u64, networ
...
-pub fn encode_lock_until_time(num: u64) -> Vec<u8> {
-    let output = OutputTimeLock::UntilTime(BlockTimestamp::from_int_seconds(num));
+pub fn encode_lock_until_time(timestamp_since_epoch_in_seconds: u64) -> Vec<u8> {
+    let output = OutputTimeLock::UntilTime(BlockTimestamp::from_int_seconds(
+        timestamp_since_epoch_in_seconds,
+    ));
     output.encode()
 }
 
+/// Given a block height, this function returns the the output timelock which is used in
```
Comment (OBorce) [inline wasm-wrappers/src/lib.rs:360]: same the the

File: wasm-wrappers/src/lib.rs [hunk 6]
```diff
@@ -328,28 +384,34 @@ pub fn encode_output_lock_then_transfer(
...
+pub fn encode_output_coin_burn(amount: &str) -> Result<Vec<u8>, Error> {
     let amount = parse_amount(amount)?;
 
     let output = TxOutput::Burn(Coin(amount));
     Ok(output.encode())
 }
 
+/// Given a pool id as string, an owner address and a network type (mainnet, testnet, etc),
+/// this function returns an output (as bytes) to create a delegation to the given pool.
+/// The owner address is the address that is authorized to withdraw from that pool.
```
Comment (OBorce) [inline wasm-wrappers/src/lib.rs:398]: withdraw from that delegation?

File: wasm-wrappers/src/lib.rs (hunk 4 above)
Comment (TheQuantumPhysicist) [reply wasm-wrappers/src/lib.rs:350]: Done

File: wasm-wrappers/src/lib.rs (hunk 3 above)
Comment (TheQuantumPhysicist) [reply wasm-wrappers/src/lib.rs:342]: Done

File: wasm-wrappers/src/lib.rs (hunk 2 above)
Comment (TheQuantumPhysicist) [reply wasm-wrappers/src/lib.rs:340]: Done

File: wasm-wrappers/src/lib.rs (hunk 5 above)
Comment (TheQuantumPhysicist) [reply wasm-wrappers/src/lib.rs:360]: Done

File: wasm-wrappers/src/lib.rs (hunk 1 above)
Comment (TheQuantumPhysicist) [reply wasm-wrappers/src/lib.rs:66]: Done. Thanks for proof reading everything!

**Pull request merged**